base/src/iosMain/kotlin/text/KalugaDateFormatter.kt

```diff
@@ -134,12 +137,12 @@ actual class DateFormatter private constructor(private val format: NSDateFormatt
         get() = format.PMSymbol
         set(value) { format.PMSymbol = value }
 
-    actual fun format(date: Date): String = format.stringFromDate(date.date)
-    actual fun parse(string: String): Date? {
+    actual fun format(date: KalugaDate): String = format.stringFromDate((date as DefaultKalugaDate).date)
```

Comment:
what changed here?

Reply:
Looked a bit more, but this is a poor API. `stringFromDate` should be able to handle a `KalugaDate`.

Reply:
[99ad364](https://github.com/splendo/kaluga/pull/465/commits/99ad3646d35141038301616757893163b5f7a55f)